src/components/modules/shared/ArticleCountOptOutOverlay.tsx

```diff
@@ -76,15 +76,15 @@ const overlayBody = css`
 
 const overlayCtaContainer = css`
     margin-top: ${space[3]}px;
+    display: flex;
+    flex-direction: row;
+
     > * + * {
-        margin-top: ${space[3]}px;
```

Comment:
why did we have margin-top?

Reply:
hmmmm I'm not sure - that css wasn't actually doing anything for some reason.

Reply:
ahhh! actually - I think the ctas used to be stacked on top of each other at mobile, just for some other reason that wasn't happening now

Reply:
can stacking still happen at narrow widths? Or does flex prevent that now?

Reply:
Right now they wont stack if you get too small e.g
<img width="409" alt="Screenshot 2020-11-25 at 13 39 48" src="https://user-images.githubusercontent.com/17720442/100234988-c0352000-2f23-11eb-9e17-fb199e4ea405.png">

Reply:
ok, that would be a ridiculously narrow screen so let's not worry about it!